.circleci/config.yml

```diff
@@ -834,7 +843,7 @@ workflows:
           aws-secret-access-key: DEV_AWS_SECRET_ACCESS_KEY
           shuttle-env: staging
           requires:
-            - approve-push-unstable
+            - approve-build-images-unstable
```

Comment:
Nit: it does both build and push.

Reply:
Good catch! Done. I added an approve step for building the images on production, but we don't use it, since merging something on `production` is intentional, only for a release, so approving the images build & push isn't needed.